src/codec_svt.c

```diff
@@ -207,6 +207,7 @@ static avifBool svtCodecEncodeFinish(avifCodec * codec, avifCodecEncodeOutput *
     input_buffer.p_app_private = NULL;
     input_buffer.flags = EB_BUFFERFLAG_EOS;
     input_buffer.p_buffer = NULL;
+    input_buffer.metadata = NULL;
```

Comment:
Do you think we should set input_buffer.size after line 203?

```
    input_buffer.size = sizeof(EbBufferHeaderType);
```

Reply:
From what I understand based on the sample app, .size does not need to be set if sending an EOS